### DISCUSSION TITLE
nameserver是否能开启acl认证

### DISCUSSION THREAD

**Discussion by iamqq23ue** [apache/rocketmq#7072] [Q&A]
nameserver上没有找到启用认证的方法。
看见cve-2023-37582建议打开认证，实际上，没有认证的方法吧

**Comment by onejimmyboy** [comment]
没有吧，这个漏洞，最终是在5.1.3版本修复的，

**Reply by iamqq23ue** [reply]
对，漏洞解决了。不过如果不能开启认证的话，以后还可能有其他漏洞，所以咨询下是否能开启认证。
